src/models/bundle-descriptor.ts

```diff
@@ -1,19 +1,35 @@
 import { MicroFrontendStack, MicroserviceStack } from './component'
 
 export type EnvironmentVariable =
-  | {
-      name: string
-      value: string
-    }
-  | {
+  | SimpleEnvironmentVariable
+  | SecretEnvironmentVariable
+
+export type SimpleEnvironmentVariable = {
+  name: string
+  value: string
+}
+
+export type SecretEnvironmentVariable = {
+  name: string
+  secretKeyRef: {
+    name: string
+    key: string
+  }
+}
+
+export type SecretEnvironmentVariableYaml = {
```

Comment:
This should not be here but in yaml-bundle-descriptor, no?

Reply:
done